### DISCUSSION TITLE
无法访问开往页面

### DISCUSSION THREAD

**Discussion by BCTX2010** [travellings-link/travellings#1621] [寻求帮助] [answered]
我是新白菜频道的站长，杭州滨江,https://www.travellings.cn/挂了

**Comment by BCTX2010** [comment]
@volfclub @linlinzzo @qystudio0 @HAHH9527 @Jiaocz @essesoul @A7103 @LeonShaw @Cubik65536 @rabbitxuanxuan @baipin @Big-Cake-jpg

**Comment by linlinzzo** [comment, marked as answer]
您好，目前开往域名 `travellings.cn` 正在进行 ICP 备案主体的变更，目前处于关站状态。您可以通过访问 `travellings.xiaozonglin.cn` 来临时开往新的网站。我们会在网站恢复后第一时间通知您。抱歉给您造成不便。

**Comment by BCTX2010** [comment]
好的，谢谢，接下来也会继续支持“开往”项目

**Comment by linlinzzo** [comment]
开往的部分服务已恢复，您可以尝试访问 https://www.travellings.cn/ 。如果您发现开往的服务问题或者想对开往提出意见，欢迎您再次使用 Discussion。
